core/server/services/webhooks/trigger.js

```diff
@@ -40,22 +40,13 @@ function makeRequests(webhooksCollection, payload, options) {
                     common.logging.warn(`Unable to destroy webhook ${webhookId}`);
                 });
             }
-            let lastTriggeredError = err.statusCode ? '' : `Failed to send request to ${targetUrl}`;
+            let lastTriggeredError = err.statusCode ? '' : `Request failed: ${err.code || ''}`;
```

Comment:
Why are we getting rid of `targetUrl`? Think that might be quite useful information in case we decide to show this message to the client :thinking:

Reply:
>  targetUrl is already known to the client because this error message gets stored on the webhook model instance.

👍 This was the reason, along with fact error code is the new extra data for failure in case of no status. 

> we should probably clear the error message if the target_url field changes
🤔 Actually this is an interesting case for all `last_triggered_*` fields, including `last_triggered_at` and `last_triggered_status`, as they are all implicitly tied to `target_url` currently. Not 100% if its ideal to clear them all on `target_url` change in short term as they do store the right `last_triggered` info even if url changed which can be useful. Long term it should be fixed with separate trigger table discussed before.

Reply:
@kevinansfield @gargol Does this sound allright? ^ 😃

Reply:
@rishabhgrg yeah, I would also be inclined to keep it as-is for now and not clear them on `target_url` change. Seems like a premature optimisation for an edge case 😄

Reply:
@rishabhgrg :+1: as long as `last_triggered_*` data that is associated with updated `target_url` doesn't cause any trouble (we don't seem to have any logic dependant on it) wouldn't change either :)

Reply:
Cool thanks, there isn't any logic dependent on it which should cause trouble, so if the rest looks ok then will merge this today 😄 👍